app/utils/draft/index.ts

```diff
@@ -3,11 +3,13 @@
 
 import {Alert, type AlertButton} from 'react-native';
 
+import {parseMarkdownImages, removeDraft, updateDraftMarkdownImageMetadata, updateDraftMessage} from '@actions/local/draft';
 import {General} from '@constants';
 import {CODE_REGEX} from '@constants/autocomplete';
 import {t} from '@i18n';
 
 import type {IntlShape, MessageDescriptor} from 'react-intl';
+import type {Swipeable} from 'react-native-gesture-handler';
```

Comment:
Deprecated notice.  Mentioned in another file.

Reply:
Removed it, and replaced it with `ReanimatedSwipeable`